pkg/sdn/plugin/node.go

```diff
@@ -272,11 +272,13 @@ func (node *OsdnNode) Start() error {
 				continue
 			}
 			if vnid, err := node.policy.GetVNID(p.Namespace); err == nil {
-				node.policy.RefVNID(vnid)
+				node.policy.EnsureVNIDRules(vnid)
 			}
 		}
 	}
 
+	go kwait.Forever(node.syncVNIDRules, 24*time.Hour)
```

Comment:
No. Note that pod/VNID rules are very different in 1.5 than in earlier releases.

The rules referring to IP addresses aren't the ones that are being left behind. The stale rules only refer to VNIDs. So in your example, after creating the first pod, there will be two rules, one that says "if the destination IP is 10.1.11.17 then the destination VNID is 2000" and one that says "if the source VNID is 2000 and the destination VNID is 2000 then allow the traffic". After deleting the pod, the first rule will be deleted but the second will not. If there are no longer any pods with VNID 2000 on the node, then the VNID rule is now useless, but it's also harmless.